client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -686,6 +686,23 @@
                             <span data-ng-if="!ola.isActivated()"
                                   data-translate="server_ola_inactive"></span>
                         </oui-tile-description>
+                        <oui-action-menu
+                            aria-label="{{:: 'menu_action_label' | translate:{ t0: ('server_traffic' | translate) } }}"
+                            data-align="end"
+                            data-compact>
+                            <oui-action-menu-item
+                                data-ng-if="!ola.isActivated()"
+                                data-text="{{:: 'server_ola_activate' | translate }}"
+                                aria-label="{{:: 'server_ola_activate' | translate }}"
```

Comment:
Do we have to say it twice though ? 🤔

Reply:
I don't think the translation is modified by the component. We could remove it

Reply:
I'll check if we have something to enhance it, or else I'll remove it